.pre-commit-config.yaml

```
@@ -1,31 +1,55 @@
+    rev: v1.33.0
+    hooks:
+      - id: yamllint
+        args: [--strict]
+
+  - repo: https://github.com/python-poetry/poetry
+    rev: 1.7.1
+    hooks:
+      - id: poetry-check
+      - id: poetry-lock
+      - id: poetry-export
+        args: ["-f", "requirements.txt", "-o", "requirements.txt"]
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)